azure-spring-boot/src/main/java/com/microsoft/azure/keyvault/spring/KeyVaultEnvironmentPostProcessorHelper.java

```diff
@@ -21,42 +21,46 @@
 import org.springframework.util.ClassUtils;
 
 import java.util.HashMap;
+import java.util.Optional;
 
 class KeyVaultEnvironmentPostProcessorHelper {
 
     private final ConfigurableEnvironment environment;
     private final TelemetryProxy telemetryProxy;
 
-    public KeyVaultEnvironmentPostProcessorHelper(ConfigurableEnvironment environment) {
+    public KeyVaultEnvironmentPostProcessorHelper(final ConfigurableEnvironment environment) {
         this.environment = environment;
         this.telemetryProxy = new TelemetryProxy(this.allowTelemetry(environment));
     }
 
     public void addKeyVaultPropertySource() {
-        final String clientId = getProperty(environment, Constants.AZURE_CLIENTID);
-        final String clientKey = getProperty(environment, Constants.AZURE_CLIENTKEY);
-        final String vaultUri = getProperty(environment, Constants.AZURE_KEYVAULT_VAULT_URI);
-
-        final long timeAcquiringTimeoutInSeconds = environment.getProperty(
+        final String clientId = getProperty(this.environment, Constants.AZURE_CLIENTID);
+        final String clientKey = getProperty(this.environment, Constants.AZURE_CLIENTKEY);
+        final String vaultUri = getProperty(this.environment, Constants.AZURE_KEYVAULT_VAULT_URI);
+        final Long refreshInterval = Optional.ofNullable(
+                this.environment.getProperty(Constants.AZURE_KEYVAULT_REFRESH_INTERVAL))
+                .map(Long::valueOf)
+                .orElse(1800000L);
```

Comment:
@dcherix Thank you very much for opening PR. Can we move this default refresh interval `1800000L` to the Constants class for reuse?